Update teams-powershell-move-from-sfbo.md (#6729)
MicrosoftDocs/OfficeDocs-SkypeForBusiness: Teams/teams-powershell-move-from-sfbo.md

# Move from Skype for Business Online Connector to the Teams PowerShell module

> To move from using Skype for Business Online Connector to the Teams PowerShell module to manage Teams, you'll need to update existing PowerShell scripts. This article explains how to do this. Note that using Teams PowerShell module you can manage both Skype for Business Online and Teams users.

Reviewer: ```suggestion
To move from using Skype for Business Online Connector to the Teams PowerShell module to manage Teams, you'll need to update existing PowerShell scripts. This article explains how to do this. Note that by using Teams PowerShell module you can manage both Skype for Business Online and Teams users.
```

1. Install the latest Teams PowerShell module. For steps, see [Install Microsoft Teams Powershell](teams-powershell-install.md).

> 2. Uninstall Skype For Business Online Connector. To do this, in Control Panel, go **Programs and Features**, select **Skype for Business Online, Windows PowerShell Module**, and then select **Uninstall**.

Reviewer: ```suggestion
2. Uninstall Skype For Business Online Connector. To do this, in Control Panel, go to **Programs and Features**, select **Skype for Business Online, Windows PowerShell Module**, and then select **Uninstall**.
```

3. 
   | Original                              | Change To                     | Additional Comments                                             |
   |-------------------------------------- |-------------------------------|-----------------------------------------------------------------|
   | Import-Module SkypeOnlineConnector    | Import-Module MicrosoftTeams  |                                                                 |
   | Import-Module LyncOnlineConnector     | Import-Module MicrosoftTeams  |                                                                 |

>    | $session = New-CsOnlineSession <br> Import-PsSession -Session $session    | Connect-MicrosoftTeams        | using New-CsOnlineSession and importing session is optional at this time and will be deprecated in the future                  |

Reviewer: ```suggestion
   | $session = New-CsOnlineSession <br> Import-PsSession -Session $session    | Connect-MicrosoftTeams        | Using New-CsOnlineSession and importing session is optional at this time and will be deprecated in the future.                  |
```

>    | Enable-CsOnlinesessionForReconnection | -Remove it-                   | reconnections are now made automatic and this cmdlet is no longer needed and thus removed from module  |

Reviewer: ```suggestion
   | Enable-CsOnlinesessionForReconnection | -Remove it-                   | Reconnections are now made automatically and this cmdlet is no longer needed, thus it is removed from the module.  |
```
